website/pages/docs/concepts/seal.mdx

```diff
@@ -84,80 +84,129 @@ to decrypt the master key at startup and unseal automatically.
+
+~> **NOTE**: Seal migration from KMS seal to Shamir seal is not currently
+supported when using Vault Enterprise. We plan to support this officially in a
+future release.
+
+~> **NOTE**: Seal migration from KMS seal to KMS seal of same kind is not
+currently supported when using Vault Enterprise. We plan to support this
+officially in a future release.
+
+### Migration post Vault 1.4.0
 
-~> **NOTE**: The migration operation will require both seals to be available
-during the migration. For example, a migration from a cloud KMS seal to
-Shamir will require that the cloud KMS be accessible during the migration.
+These steps are common for seal migrations between any supported kinds and for
+any storage backend.
 
-### Migration From Shamir to Auto Unseal
+1. Take the standby nodes down, one node at a time and update the [seal
+configuration](/docs/configuration/seal). If the migration is from Shamir seal
+to KMS seal, add the desired new KMS seal block to the config. If the migration
+is from KMS seal to Shamir seal, add `disabled = "true"` to the old seal block.
+If the migration is from KMS seal to another KMS seal, add `disabled = "true"`
+to the old seal block and add the desired new KMS seal block.
 
-To migrate from Shamir keys to Auto Unseal, take your server cluster offline and update
-the [seal configuration](/docs/configuration/seal) with the appropriate seal
-configuration. Bring your server back up and leave the rest of the nodes offline if
-using multi-server mode, then run the unseal process with the `-migrate` flag and bring
-the rest of the cluster online.
+2. Bring the standby nodes back up one at a time and run the unseal command on
+each by supplying the `-migrate` flag. Supply Shamir unseal keys if old seal was
+Shamir, which will be migrated as the recovery keys for the KMS seal. Supply
+recovery keys if the old seal is one of KMS seals, which will be migrated as the
+recovery keys of the new KMS seal or as Shamir unseal keys if the new seal is
+Shamir.
 
-All unseal commands must specify the `-migrate` flag. Once the required threshold of
-unseal keys are entered, unseal keys will be migrated to recovery keys.
+3. Stop the active node. One of the standby nodes will become the active node
+and perform the migration. If possible, monitor the server log in the active
+node to witness the completion of the seal migration process.
 
-```
-$ vault operator unseal -migrate
-```
+4. Seal migration is now completed. Update the config of the old active node
+(that is still down) to use the new seal blocks (completely unaware of the old
+seal type) and bring it up. It will be auto-unsealed if the new seal is one of the
+KMS seals or will require unseal keys if the new seal is Shamir.
 
-### Migration From Auto Unseal to Shamir
+5. At this point, config files of all the nodes can be updated to only have the
+new seal information. Standby nodes can be restarted right away and the active
+node can be restarted upon a leadership change.
```

Comment:
Should probably make sure all sealwrapped data is finished re-wrapping before doing this step.

Reply:
I've updated step 3 to cover this. Let me know what you think of that.